python/cuda_parallel/cuda/parallel/experimental/_bindings.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+#
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import os
+import shutil
+import importlib
+import ctypes
+
+from . import _cccl as cccl
+
+
+def _get_cuda_path():
+    cuda_path = os.environ.get("CUDA_PATH", "")
+    if os.path.exists(cuda_path):
+        return cuda_path
+
+    nvcc_path = shutil.which("nvcc")
+    if nvcc_path is not None:
+        return os.path.dirname(os.path.dirname(nvcc_path))
+
+    default_path = "/usr/local/cuda"
+    if os.path.exists(default_path):
+        return default_path
+
+    return None
+
+
+_bindings = None
+_paths = None
+
+
+def get_bindings():
+    global _bindings
+    if _bindings is None:
```

Comment:
I'd do this to avoid having to indent the much longer block:
```
if _bindings is not None:
    return _bindings
```

Reply:
Good idea - fixed.